VideoLocker/res/drawable/list_item_selector.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@color/edx_grayscale_neutral_light" android:state_pressed="true"/>
-    <item android:drawable="@color/white"/>
```

Comment:
Better to leave this and make it `@android:color/transparent` to be on the safe side, since some versions of Android (due to a bug?) will render this drawable as black if there is no matching selector.